catalog/video/video-mpegts-stream/pom.xml

```diff
@@ -25,6 +25,23 @@
     <artifactId>video-mpegts-stream</artifactId>
     <name>Alliance :: Video :: Stream</name>
     <packaging>bundle</packaging>
+
+    <dependencyManagement>
```

Comment:
❓ Did we have to use `<dependencyManagement>` to get this to work? Could we have used a simple `<dependency>` without a `<version>`, scoped for test and with the `<exclusion>` instead?